workshops/templates/workshops/airport.html

```diff
@@ -11,7 +11,15 @@
 </table>
 
 <div class="clearfix">
+  {% if perms.workshops.change_airport %}
   <p class="edit-object pull-left"><a href="{% url 'airport_edit' airport.iata %}" class=" btn btn-primary">Edit</a></p>
+  {% else %}
+  <p class="edit-object pull-left"><a href="{% url 'airport_edit' airport.iata %}" class=" btn btn-primary disabled">Edit</a></p>
+  {% endif %}
```

Comment:
I'm against these changes.

Here are the reasons:

1. These are logical paths in the templates, and it's easier to change them in future if you want to take a different action if some conditions are (not) met
2. You're easier to spot these paths if they are marked like: `{% if %} \n blah \n {% endif %}`
3. Highlighting Django template / Jinja syntax within HTML attributes is flawed (another reason why it's so hard to spot these inline, in-attr conditionals)
4. Too long lines (even with wrapping) make it harder to understand the code

So, generally, I'm opting for code duplication in sake of its' clearness.